src/locales/en.json

```diff
@@ -238,13 +239,18 @@
     "basic_mask": "Multi-layer cloth mask with tight, secure fit",
     "basic_mask_short": "Cloth, thick and snug",
     "car_windows_down": "A moving car with the windows rolled down",
+    "car_windows_down_breakdown": "for car",
```

Comment:
for airflow from being in a car with the windows down

Reply:
that's really long, it'll take up most of the line